store/extensions/Q-Wired-Page/extension.json

```
@@ -0,0 +1,40 @@
+{
+  "title": "Q-Wired-Page",
+  "description": "Removes the parent variable menu item and adds the children to the Wired menu.",
+  "authors": [
+    {
+      "name": "MrGeneralQ",
+      "hotel": ".nl",
+      "username": "MrGeneralQ",
+      "discord": "MrGeneralQ",
+      "twitter": ""
+    }
+  ],
+  "version": "1.0",
+  "categories": ["Others"],
```

```suggestion
  "categories": ["Usability"],
```
This fits under usability: https://github.com/sirjonasxx/G-ExtensionStore/blob/repo/1.5.3/store/config.json#L28-L32